subiquitycore/ui/utils.py

```diff
@@ -180,3 +188,8 @@ class Color:
 
     """
     pass
+
+
+def connect_signal(obj, name, callback, user_arg=None, weak_args=None, user_args=None):
+    obj = getattr(obj, 'base_widget', obj)
+    return _connect_signal(obj, name, callback, user_arg, weak_args, user_args)
```

Comment:
Hm.... I have an iffy feeling about this. Can we not reuse the same name? Or e.g. somehow monkey-patch urwid to do this. I fear we will make the mistake somewhere of using `urwid.connect_signal` instead of this =/ it would be ideal to not have this at all.... and just use the stock one, somehow.

Reply:
Yeah I don't like this greatly either. We could just change the places where it is needed to do connect_signal(w.base_widget) instead but that seems error-prone and annoying too.

Reply:
I just removed the wrapper in favour of using on_press everywhere.